bot/app/database/db_objects.py

```diff
@@ -53,10 +55,12 @@ def insert_event(self):
         conn = sqlite3.connect(DB_NAME)
         curs = conn.cursor()
 
+        event_id = self.get_max_event()
+
         try:
             curs.execute("""
-                insert into Events values (?,?,?,?,?,?,?)""",
-                (self.name, self.date, self.time, self.admission, self.description, \
+                insert into Events values (?,?,?,?,?,?,?,?)""",
+                (event_id, self.name, self.date, self.time, self.admission, self.description, \
```

Comment:
Since you're using `eventID integer primary key`, which is auto-increment, the first value should always be `NULL`.

Reply:
fixed